LiskHQ/lisk-sdk
# Save P2P secret in the database - Closes #4040

### What was the problem?
P2P secret was not saved in the database and it was changing everytime

### How did I solve it?
- Create database
- Save if the secret doesnt exist and if exist use the secret

### How to manually test it?
Start the node and see if the value is in `network_info` table, and restart the node and see secret doesn't change

### Review checklist

- [ ] The PR resolves #4040
